Re-enable and tweak sms-gateway e2e test

dianabarsan opened the pull request:
# Description

Fixes race condition in ngx-bootstrap related to rendering forms dropups. 
Fixes race condition where queued component tasks can change actionbars after the components are destroyed.
Re-orders e2e tests so they run alphabetically. (this exposed all bugs that needed to be fixed). 
Tries to stabilize message-duplicates e2e test

https://github.com/medic/angular10-migration/issues/155
https://github.com/medic/angular10-migration/issues/154
https://github.com/medic/angular10-migration/issues/90
https://github.com/medic/angular10-migration/issues/158

# Code review checklist
<!-- Remove or comment out any items that do not apply to this PR; in the remaining boxes, replace the [ ] with [x]. -->
- [x] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [x] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [x] Tested: Unit and/or e2e where appropriate
- [x] Internationalised: All user facing text
- [x] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

dianabarsan commented:
hey @latin-panda 
Do you mind checking out these 2 bug fixes + an e2e test fix? Thanks!

latin-panda reviewed (commented):
Great job! I added a question, but all the rest looks good.

dianabarsan commented on webapp/src/ts/modules/messages/messages.component.ts:
```diff
@@ -140,6 +149,7 @@ export class MessagesComponent implements OnInit, OnDestroy {
   }
 
   updateConversations({merge = false} = {}) {
+    console.log(this.destroyed, 'destroyed');
```
wops!

dianabarsan replied:
thanks for noticing!!!

latin-panda reviewed (approved):
Excellent work! ✨

dianabarsan commented:
hey @latin-panda 
I added and then reverted a bunch of stuff, so the PR is in the same state as you reviewed it with one exception: I reverted the change I made to the config file where I reordered the tests. For some reason I don't yet understand, it made the tests super unstable (I had consistent webdriver crashes). So I've decided I'll try that out in another PR.
